### PR TITLE
Add aarch64, ppc64le, and s390x builds to GitHub actions

### PR DESCRIPTION
**Pull request by astrofrog**
This makes use of the [run-on-arch](https://github.com/uraimo/run-on-arch-action) action to run the test suite with minimal dependencies (for now) on aarch64, ppc64le, and s390x.

The builds are very slow, usually between 1 and 2 hours each **but** given that this only done once per week, I don't think it's actually a problem.

The builds on Travis used the system libraries bit I think it might be better to use the bundled ones to make sure we compile them right on those systems?

There are some real PPC failures that have been identified by the ppc64le build (#11025, #11026, and #11027). We didn't have a PPC build before, just aarch64 and s390x.

Not sure if we need to backport this or not - what do others think?

⚠️ there is a temporary commit that needs to be removed prior to merging! ⚠️

### CODE REVIEW

**Comment by mhvk** [comment]
@astrofrog - this looks nice. On the system libraries, I think it is important that we test them *somewhere*, and ideally in a run that also does other stuff. I thought the other archs were useful for this, since those essentially are used only through Debian/Ubuntu anyway, so we may as well use their packages.

But more importantly, you note in the PR that the runs are so slow because they are run on an emulator. I am not sure that in that case there is much point - my sense would be to just keep travis cron jobs instead (this despite rather disliking having multiple systems).

**Comment by bsipocz** [comment]
> But more importantly, you note in the PR that the runs are so slow because they are run on an emulator. I am not sure that in that case there is much point - my sense would be to just keep travis cron jobs instead (this despite rather disliking having multiple systems).

@mhvk - sadly it's not really an option any more, travis stopped working 2 days ago for all astropy repos, thus the recent hard work of moving everything to other platforms.

**Comment by bsipocz** [comment]
As of backport: yes, I think it would be useful to have the test run on 4.0.x, but it cannot be a backport but a separate PR to that branch. Also, I think it's worth to have these tests run on 4.2.x. For the latter, it can be a backport, but there I could bring it to the normal matrix rather than leave it in cron given the branch is low traffic and it's best to see all passes when packports are done, but no need for a cronjob there.

**Comment by bsipocz** [comment]
> The builds are very slow, usually between 1 and 2 hours each but given that this only done once per week, I don't think it's actually a problem.

While it's not ideal, I think that's OK for this infrequent cron. It would be nice to have the cron jobs report to slack though.

**Comment by mhvk** [comment]
Ouch, were we somehow kicked off travis?

Anyway, in that case :+1: to this effort. I guess this will catch most bugs (not that we've had many) and in the end we can still count on getting reports from Debian using real hardware.

**Comment by bsipocz** [comment]
@mhvk - long story short (more details will come to astropy-dev in an email, soonish): their policy toward open source changed at the beginning of the month, effective immediately, and we run out of the buffer credit they gave us very quickly (all repos in the org use the same resource): https://blog.travis-ci.com/2020-11-02-travis-ci-new-billing

**Comment by astrofrog** [comment]
@mhvk - I've now tried to replicate as close as possible to what you had originally - hopefully this will also speed up the builds

**Comment by mhvk** [comment]
Thanks, @bsipocz, given that notice there indeed doesn't seem much future for astropy on travis. Of course, someone has to pay the bills for those CI computers, so I suspect others will eventually follow, and we may end up with some shared infrastructure via NumFocus or so.  But hopefully a few years down the line.

**Comment by mhvk** [comment]
p.s. FWIW, numpy runs just the other platforms on travis still.

File: .github/workflows/ci_cron_weekly.yml
Comment (mhvk) [inline .github/workflows/ci_cron_weekly.yml:98]: This line fails for some reason!? https://github.com/astropy/astropy/pull/11045/checks?check_run_id=1402190795#step:3:853
I can reproduce this if I create a `.sh` file with the above commands with the `#!/bin/sh` - it gets executed via `sh` rather than bash.   Does one need to install `bash`? 

Though just replacing `source tests/bin/activate` with `. tests/bin/activate` does seem to make things work (though on my machine there are then errors collecting the tests).

```
Successfully tagged run-on-arch-astropy-astropy-weekly-cron-ppc64le-buster:latest
/home/runner/work/_actions/uraimo/run-on-arch-action/v2.0.5/src/run-on-arch-commands.sh: 4: /home/runner/work/_actions/uraimo/run-on-arch-action/v2.0.5/src/run-on-arch-commands.sh: source: not found
Error: The process '/home/runner/work/_actions/uraimo/run-on-arch-action/v2.0.5/src/run-on-arch.sh' failed with exit code 127
```

Comment (mhvk) [reply .github/workflows/ci_cron_weekly.yml:98]: Note that similar lines worked fine on regular architectures over at `pyerfa` - see https://github.com/liberfa/pyerfa/pull/58 - so if it is not clear what is going wrong here, perhaps we should simply revert to using `tox` and turn one of the other tests into one that uses the system libraries?

File: .github/workflows/ci_cron_weekly.yml
Comment (astrofrog) [inline .github/workflows/ci_cron_weekly.yml:81]: @mhvk thanks for investigating! I probably need to change this then :) Will try shortly.

Comment (mhvk) [reply .github/workflows/ci_cron_weekly.yml:81]: Yes, that might just be it!

Comment (mhvk) [reply .github/workflows/ci_cron_weekly.yml:81]: `/bin/bash` should exist, since it is a Debian system

**Review by pllim — COMMENTED** [review]
For future reference, there is no need to delete `.circleci`, `ci_cron.yml`, or Azure YAML. CircleCI won't run if you do `[ci skip]` but it does not affect Actions. `ci_cron.yml` and Azure won't run anyway.

As for the actual implementation of exotic archs, I'll have to defer to @mhvk . Failure now is `error trying to exec 'cc1plus': execvp: No such file or directory`.

> runs just the other platforms on travis still

@mhvk , I think the credits apply to the entire orgs and we run a risk of other packages in the org eating up the credits unexpectedly. If there is somehow a way to tell Travis CI that we can only use the credits for certain repos, then maybe, maybe...

> Not sure if we need to backport this or not - what do others think?

@astrofrog , not sure. If we make new release on, say, LTS, is it critical for these archs to run?

**Comment by astrofrog** [comment]
@mhvk - so interestingly there are now failures due to the use of system libraries which I am not sure why we did not pick up before. Basically we use some features from WCSLIB 7 but currently WCSLIB 6 is picked up.

@olebole - is there any way to apt install WCSLIB 7 on Debian buster?

**Comment by olebole** [comment]
I could create a backport (linked to the Buster version of cfitsio) and put it to [buster-backports](https://backports.debian.org/). You would need to add the line
```
deb http://deb.debian.org/debian buster-backports main
```
to `/etc/apt/sources.list` to make use of it. Would that help you?

**Comment by astrofrog** [comment]
@olebole - yes please, that would be great.

I'm very confused as to how this ever worked before on Travis though 😕

**Comment by mhvk** [comment]
@astrofrog, @olebole - on travis, I had moved to Ubuntu focal, which likely had newer packages then buster. I don't know how the choice of distribution is made in Actions, but is there perhaps an option to run Debian testing?

**Comment by olebole** [comment]
Ubuntu Focal (20.04 LTS) has wcslib-7.1, which should be compatible.
I now also uploaded wcslib-7.3.1 to Debian backports; within a few days it should appear there (pending review of our ftp-masters).
Using Debian testing for CI test would ofcourse be great, it would help me a lot! However, I didn't get into Github Actions yet...

**Comment by astrofrog** [comment]
For the arches in this PR we are limited to specific distributions - see https://github.com/marketplace/actions/run-on-architecture for more details. I decided on buster as the latest supported Debian version there.

**Comment by astrofrog** [comment]
@olebole - thank you for the backport! I'll update this PR in a few days.

**Comment by olebole** [comment]
I am very happy about Buster as well -- this gives me important information about backports compatibility, which makes backporting astropy simpler in future. Just drop me a note when you need other backports.

**Comment by mhvk** [comment]
plan sounds good (for the record, ubuntu20.04 is focal).

**Comment by astrofrog** [comment]
@olebole - just to check, could you put a comment here once WCSLIB 7 should be in buster-backports?

**Comment by olebole** [comment]
Sure. It is still [in the ftpmaster queue](https://ftp-master.debian.org/new/wcslib_7.3.1%2Bds-1~bpo10%2B1.html), since it has to go through a review for the first time. They are quite quick in the moment, so I expect it within a few days. I'll put a note here.

**Comment by olebole** [comment]
@astrofrog The package was [accepted today](https://tracker.debian.org/news/1195673/accepted-wcslib-731ds-1bpo101-source-amd64-all-into-buster-backports-buster-backports/) and should pop up in [buster-backports](https://packages.debian.org/buster-backports/wcslib-dev). It may take some hours until it is propagated to all mirrors, and [built for all archs](https://buildd.debian.org/status/package.php?p=wcslib&suite=buster-backports).

**Comment by astrofrog** [comment]
@olebole - thanks!

**Comment by astrofrog** [comment]
@olebole - I think I'm doing something silly but I can't get apt to find WCSLIB 7:

```
$ docker run -it debian:buster 

root@5b27be39eb3a:/# echo "deb http://deb.debian.org/debian buster-backports main" >> /etc/apt/sources.list

root@5b27be39eb3a:/# apt-get update
Get:1 http://security.debian.org/debian-security buster/updates InRelease [65.4 kB]
Get:2 http://deb.debian.org/debian buster InRelease [121 kB]
Get:3 http://deb.debian.org/debian buster-updates InRelease [51.9 kB]
Get:4 http://deb.debian.org/debian buster-backports InRelease [46.7 kB]
Get:5 http://security.debian.org/debian-security buster/updates/main amd64 Packages [251 kB]
Get:6 http://deb.debian.org/debian buster/main amd64 Packages [7906 kB]
Get:7 http://deb.debian.org/debian buster-updates/main amd64 Packages [7856 B]
Get:8 http://deb.debian.org/debian buster-backports/main amd64 Packages [378 kB]
Fetched 8829 kB in 4s (2416 kB/s)                       
Reading package lists... Done

root@5b27be39eb3a:/# apt-get install wcslib-dev
Reading package lists... Done
Building dependency tree       
Reading state information... Done
The following additional packages will be installed:
  fontconfig-config fonts-dejavu-core giza-dev libbsd0 libcairo2 libcpgplot0 libexpat1 libfontconfig1 libfreetype6 libgfortran5 libgiza0 libpgplot0
  libpgsbox6 libpixman-1-0 libpng16-16 libquadmath0 libwcs6 libx11-6 libx11-data libxau6 libxcb-render0 libxcb-shm0 libxcb1 libxdmcp6 libxext6
  libxrender1 sensible-utils ucf
Suggested packages:
  wcslib-doc
The following NEW packages will be installed:
  fontconfig-config fonts-dejavu-core giza-dev libbsd0 libcairo2 libcpgplot0 libexpat1 libfontconfig1 libfreetype6 libgfortran5 libgiza0 libpgplot0
  libpgsbox6 libpixman-1-0 libpng16-16 libquadmath0 libwcs6 libx11-6 libx11-data libxau6 libxcb-render0 libxcb-shm0 libxcb1 libxdmcp6 libxext6
  libxrender1 sensible-utils ucf wcslib-dev
0 upgraded, 29 newly installed, 0 to remove and 1 not upgraded.
Need to get 7147 kB of archives.
After this operation, 21.4 MB of additional disk space will be used.
Do you want to continue? [Y/n] 
Get:1 http://security.debian.org/debian-security buster/updates/main amd64 libfreetype6 amd64 2.9.1-3+deb10u2 [380 kB]
Get:2 http://deb.debian.org/debian buster/main amd64 sensible-utils all 0.0.12 [15.8 kB]
Get:3 http://deb.debian.org/debian buster/main amd64 ucf all 3.0038+nmu1 [69.0 kB]
Get:4 http://deb.debian.org/debian buster/main amd64 fonts-dejavu-core all 2.37-1 [1068 kB]
Get:5 http://deb.debian.org/debian buster/main amd64 fontconfig-config all 2.13.1-2 [280 kB]
Get:6 http://deb.debian.org/debian buster/main amd64 libexpat1 amd64 2.2.6-2+deb10u1 [106 kB]
Get:7 http://deb.debian.org/debian buster/main amd64 libpng16-16 amd64 1.6.36-6 [292 kB]
Get:8 http://deb.debian.org/debian buster/main amd64 libfontconfig1 amd64 2.13.1-2 [346 kB]
Get:9 http://deb.debian.org/debian buster/main amd64 libpixman-1-0 amd64 0.36.0-1 [537 kB]
Get:10 http://deb.debian.org/debian buster/main amd64 libxau6 amd64 1:1.0.8-1+b2 [19.9 kB]
Get:11 http://deb.debian.org/debian buster/main amd64 libbsd0 amd64 0.9.1-2 [99.5 kB]
Get:12 http://deb.debian.org/debian buster/main amd64 libxdmcp6 amd64 1:1.1.2-3 [26.3 kB]
Get:13 http://deb.debian.org/debian buster/main amd64 libxcb1 amd64 1.13.1-2 [137 kB]
Get:14 http://deb.debian.org/debian buster/main amd64 libx11-data all 2:1.6.7-1+deb10u1 [294 kB]
Get:15 http://deb.debian.org/debian buster/main amd64 libx11-6 amd64 2:1.6.7-1+deb10u1 [757 kB]
Get:16 http://deb.debian.org/debian buster/main amd64 libxcb-render0 amd64 1.13.1-2 [109 kB]
Get:17 http://deb.debian.org/debian buster/main amd64 libxcb-shm0 amd64 1.13.1-2 [99.2 kB]
Get:18 http://deb.debian.org/debian buster/main amd64 libxext6 amd64 2:1.3.3-1+b2 [52.5 kB]
Get:19 http://deb.debian.org/debian buster/main amd64 libxrender1 amd64 1:0.9.10-1 [33.0 kB]
Get:20 http://deb.debian.org/debian buster/main amd64 libcairo2 amd64 1.16.0-4 [689 kB]
Get:21 http://deb.debian.org/debian buster/main amd64 libgiza0 amd64 1.1.0-1 [105 kB]
Get:22 http://deb.debian.org/debian buster/main amd64 libcpgplot0 amd64 1.1.0-1 [33.2 kB]
Get:23 http://deb.debian.org/debian buster/main amd64 libquadmath0 amd64 8.3.0-6 [133 kB]
Get:24 http://deb.debian.org/debian buster/main amd64 libgfortran5 amd64 8.3.0-6 [581 kB]
Get:25 http://deb.debian.org/debian buster/main amd64 libpgplot0 amd64 1.1.0-1 [43.3 kB]
Get:26 http://deb.debian.org/debian buster/main amd64 giza-dev amd64 1.1.0-1 [144 kB]
Get:27 http://deb.debian.org/debian buster/main amd64 libwcs6 amd64 6.2-2 [262 kB]
Get:28 http://deb.debian.org/debian buster/main amd64 libpgsbox6 amd64 6.2-2 [57.5 kB]
Get:29 http://deb.debian.org/debian buster/main amd64 wcslib-dev amd64 6.2-2 [379 kB]
Fetched 7147 kB in 2s (3100 kB/s)   
debconf: delaying package configuration, since apt-utils is not installed
Selecting previously unselected package sensible-utils.
(Reading database ... 6677 files and directories currently installed.)
Preparing to unpack .../00-sensible-utils_0.0.12_all.deb ...
Unpacking sensible-utils (0.0.12) ...
Selecting previously unselected package ucf.
Preparing to unpack .../01-ucf_3.0038+nmu1_all.deb ...
Moving old data out of the way
Unpacking ucf (3.0038+nmu1) ...
Selecting previously unselected package fonts-dejavu-core.
Preparing to unpack .../02-fonts-dejavu-core_2.37-1_all.deb ...
Unpacking fonts-dejavu-core (2.37-1) ...
Selecting previously unselected package fontconfig-config.
Preparing to unpack .../03-fontconfig-config_2.13.1-2_all.deb ...
Unpacking fontconfig-config (2.13.1-2) ...
Selecting previously unselected package libexpat1:amd64.
Preparing to unpack .../04-libexpat1_2.2.6-2+deb10u1_amd64.deb ...
Unpacking libexpat1:amd64 (2.2.6-2+deb10u1) ...
Selecting previously unselected package libpng16-16:amd64.
Preparing to unpack .../05-libpng16-16_1.6.36-6_amd64.deb ...
Unpacking libpng16-16:amd64 (1.6.36-6) ...
Selecting previously unselected package libfreetype6:amd64.
Preparing to unpack .../06-libfreetype6_2.9.1-3+deb10u2_amd64.deb ...
Unpacking libfreetype6:amd64 (2.9.1-3+deb10u2) ...
Selecting previously unselected package libfontconfig1:amd64.
Preparing to unpack .../07-libfontconfig1_2.13.1-2_amd64.deb ...
Unpacking libfontconfig1:amd64 (2.13.1-2) ...
Selecting previously unselected package libpixman-1-0:amd64.
Preparing to unpack .../08-libpixman-1-0_0.36.0-1_amd64.deb ...
Unpacking libpixman-1-0:amd64 (0.36.0-1) ...
Selecting previously unselected package libxau6:amd64.
Preparing to unpack .../09-libxau6_1%3a1.0.8-1+b2_amd64.deb ...
Unpacking libxau6:amd64 (1:1.0.8-1+b2) ...
Selecting previously unselected package libbsd0:amd64.
Preparing to unpack .../10-libbsd0_0.9.1-2_amd64.deb ...
Unpacking libbsd0:amd64 (0.9.1-2) ...
Selecting previously unselected package libxdmcp6:amd64.
Preparing to unpack .../11-libxdmcp6_1%3a1.1.2-3_amd64.deb ...
Unpacking libxdmcp6:amd64 (1:1.1.2-3) ...
Selecting previously unselected package libxcb1:amd64.
Preparing to unpack .../12-libxcb1_1.13.1-2_amd64.deb ...
Unpacking libxcb1:amd64 (1.13.1-2) ...
Selecting previously unselected package libx11-data.
Preparing to unpack .../13-libx11-data_2%3a1.6.7-1+deb10u1_all.deb ...
Unpacking libx11-data (2:1.6.7-1+deb10u1) ...
Selecting previously unselected package libx11-6:amd64.
Preparing to unpack .../14-libx11-6_2%3a1.6.7-1+deb10u1_amd64.deb ...
Unpacking libx11-6:amd64 (2:1.6.7-1+deb10u1) ...
Selecting previously unselected package libxcb-render0:amd64.
Preparing to unpack .../15-libxcb-render0_1.13.1-2_amd64.deb ...
Unpacking libxcb-render0:amd64 (1.13.1-2) ...
Selecting previously unselected package libxcb-shm0:amd64.
Preparing to unpack .../16-libxcb-shm0_1.13.1-2_amd64.deb ...
Unpacking libxcb-shm0:amd64 (1.13.1-2) ...
Selecting previously unselected package libxext6:amd64.
Preparing to unpack .../17-libxext6_2%3a1.3.3-1+b2_amd64.deb ...
Unpacking libxext6:amd64 (2:1.3.3-1+b2) ...
Selecting previously unselected package libxrender1:amd64.
Preparing to unpack .../18-libxrender1_1%3a0.9.10-1_amd64.deb ...
Unpacking libxrender1:amd64 (1:0.9.10-1) ...
Selecting previously unselected package libcairo2:amd64.
Preparing to unpack .../19-libcairo2_1.16.0-4_amd64.deb ...
Unpacking libcairo2:amd64 (1.16.0-4) ...
Selecting previously unselected package libgiza0:amd64.
Preparing to unpack .../20-libgiza0_1.1.0-1_amd64.deb ...
Unpacking libgiza0:amd64 (1.1.0-1) ...
Selecting previously unselected package libcpgplot0:amd64.
Preparing to unpack .../21-libcpgplot0_1.1.0-1_amd64.deb ...
Unpacking libcpgplot0:amd64 (1.1.0-1) ...
Selecting previously unselected package libquadmath0:amd64.
Preparing to unpack .../22-libquadmath0_8.3.0-6_amd64.deb ...
Unpacking libquadmath0:amd64 (8.3.0-6) ...
Selecting previously unselected package libgfortran5:amd64.
Preparing to unpack .../23-libgfortran5_8.3.0-6_amd64.deb ...
Unpacking libgfortran5:amd64 (8.3.0-6) ...
Selecting previously unselected package libpgplot0:amd64.
Preparing to unpack .../24-libpgplot0_1.1.0-1_amd64.deb ...
Unpacking libpgplot0:amd64 (1.1.0-1) ...
Selecting previously unselected package giza-dev:amd64.
Preparing to unpack .../25-giza-dev_1.1.0-1_amd64.deb ...
Unpacking giza-dev:amd64 (1.1.0-1) ...
Selecting previously unselected package libwcs6:amd64.
Preparing to unpack .../26-libwcs6_6.2-2_amd64.deb ...
Unpacking libwcs6:amd64 (6.2-2) ...
Selecting previously unselected package libpgsbox6:amd64.
Preparing to unpack .../27-libpgsbox6_6.2-2_amd64.deb ...
Unpacking libpgsbox6:amd64 (6.2-2) ...
Selecting previously unselected package wcslib-dev:amd64.
Preparing to unpack .../28-wcslib-dev_6.2-2_amd64.deb ...
Unpacking wcslib-dev:amd64 (6.2-2) ...
Setting up libexpat1:amd64 (2.2.6-2+deb10u1) ...
Setting up libpixman-1-0:amd64 (0.36.0-1) ...
Setting up libxau6:amd64 (1:1.0.8-1+b2) ...
Setting up libwcs6:amd64 (6.2-2) ...
Setting up libx11-data (2:1.6.7-1+deb10u1) ...
Setting up libquadmath0:amd64 (8.3.0-6) ...
Setting up libpng16-16:amd64 (1.6.36-6) ...
Setting up fonts-dejavu-core (2.37-1) ...
Setting up libgfortran5:amd64 (8.3.0-6) ...
Setting up sensible-utils (0.0.12) ...
Setting up libbsd0:amd64 (0.9.1-2) ...
Setting up libxdmcp6:amd64 (1:1.1.2-3) ...
Setting up libxcb1:amd64 (1.13.1-2) ...
Setting up libxcb-render0:amd64 (1.13.1-2) ...
Setting up libxcb-shm0:amd64 (1.13.1-2) ...
Setting up libfreetype6:amd64 (2.9.1-3+deb10u2) ...
Setting up ucf (3.0038+nmu1) ...
debconf: unable to initialize frontend: Dialog
debconf: (No usable dialog-like program is installed, so the dialog based frontend cannot be used. at /usr/share/perl5/Debconf/FrontEnd/Dialog.pm line 76.)
debconf: falling back to frontend: Readline
debconf: unable to initialize frontend: Readline
debconf: (Can't locate Term/ReadLine.pm in @INC (you may need to install the Term::ReadLine module) (@INC contains: /etc/perl /usr/local/lib/x86_64-linux-gnu/perl/5.28.1 /usr/local/share/perl/5.28.1 /usr/lib/x86_64-linux-gnu/perl5/5.28 /usr/share/perl5 /usr/lib/x86_64-linux-gnu/perl/5.28 /usr/share/perl/5.28 /usr/local/lib/site_perl /usr/lib/x86_64-linux-gnu/perl-base) at /usr/share/perl5/Debconf/FrontEnd/Readline.pm line 7.)
debconf: falling back to frontend: Teletype
Setting up libx11-6:amd64 (2:1.6.7-1+deb10u1) ...
Setting up libxrender1:amd64 (1:0.9.10-1) ...
Setting up fontconfig-config (2.13.1-2) ...
debconf: unable to initialize frontend: Dialog
debconf: (No usable dialog-like program is installed, so the dialog based frontend cannot be used. at /usr/share/perl5/Debconf/FrontEnd/Dialog.pm line 76.)
debconf: falling back to frontend: Readline
debconf: unable to initialize frontend: Readline
debconf: (Can't locate Term/ReadLine.pm in @INC (you may need to install the Term::ReadLine module) (@INC contains: /etc/perl /usr/local/lib/x86_64-linux-gnu/perl/5.28.1 /usr/local/share/perl/5.28.1 /usr/lib/x86_64-linux-gnu/perl5/5.28 /usr/share/perl5 /usr/lib/x86_64-linux-gnu/perl/5.28 /usr/share/perl/5.28 /usr/local/lib/site_perl /usr/lib/x86_64-linux-gnu/perl-base) at /usr/share/perl5/Debconf/FrontEnd/Readline.pm line 7.)
debconf: falling back to frontend: Teletype
Setting up libxext6:amd64 (2:1.3.3-1+b2) ...
Setting up libfontconfig1:amd64 (2.13.1-2) ...
Setting up libcairo2:amd64 (1.16.0-4) ...
Setting up libgiza0:amd64 (1.1.0-1) ...
Setting up libpgplot0:amd64 (1.1.0-1) ...
Setting up libcpgplot0:amd64 (1.1.0-1) ...
Setting up libpgsbox6:amd64 (6.2-2) ...
Setting up giza-dev:amd64 (1.1.0-1) ...
Setting up wcslib-dev:amd64 (6.2-2) ...
Processing triggers for libc-bin (2.28-10) ...

root@5b27be39eb3a:/# apt-get install wcslib    
Reading package lists... Done
Building dependency tree       
Reading state information... Done
E: Unable to locate package wcslib
```

What am I doing wrong?

**Comment by olebole** [comment]
The package is called `wcslib-dev`, and you need to specify the distribution (buster-backports). Just tested on a plain buster chroot:
```
root ~# echo deb http://deb.debian.org/debian buster-backports main >> /etc/apt/sources.list
root ~# apt-get update
[…]
root ~# apt-get install wcslib-dev/buster-backports
Reading package lists... Done
Building dependency tree       
Reading state information... Done
Selected version '7.3.1+ds-1~bpo10+1' (Debian Backports:buster-backports [amd64]) for 'wcslib-dev'
The following additional packages will be installed:
  fontconfig-config fonts-dejavu-core giza-dev libcairo2 libcpgplot0 libexpat1 libfontconfig1 libfreetype6 libgfortran5 libgiza0
  libpgplot0 libpgsbox7 libpixman-1-0 libpng16-16 libquadmath0 libwcs7 libx11-6 libx11-data libxau6 libxcb-render0 libxcb-shm0 libxcb1
  libxdmcp6 libxext6 libxrender1 ucf
Suggested packages:
  wcslib-doc
The following NEW packages will be installed:
  fontconfig-config fonts-dejavu-core giza-dev libcairo2 libcpgplot0 libexpat1 libfontconfig1 libfreetype6 libgfortran5 libgiza0
Do you want to continue? [Y/n] 
[…]
Get:25 http://deb.debian.org/debian buster-backports/main amd64 libwcs7 amd64 7.3.1+ds-1~bpo10+1 [268 kB]
Get:26 http://deb.debian.org/debian buster-backports/main amd64 libpgsbox7 amd64 7.3.1+ds-1~bpo10+1 [60.8 kB]
Get:27 http://deb.debian.org/debian buster-backports/main amd64 wcslib-dev amd64 7.3.1+ds-1~bpo10+1 [386 kB]
```

**Comment by astrofrog** [comment]
Note for other maintainers: still waiting on s390x then will remove the commit disabling the cron

**Comment by olebole** [comment]
Now, the s390x, ppl64el, arm64 build should be there, says [the log](https://buildd.debian.org/status/package.php?p=wcslib&suite=buster-backports).

**Comment by astrofrog** [comment]
Thanks!

**Comment by astrofrog** [comment]
Great, aarch64 and s390x passed! PPC failed but those are known issues: https://github.com/astropy/astropy/issues?q=is%3Aopen+is%3Aissue+label%3Aplatform-ppc

Here's a screenshot of the builds:

Screenshot 2020-11-30 at 20 33 53



I'll now remove the temporary commit.

**Comment by astrofrog** [comment]
This is ready for review - note that we don't need to wait for the CI to pass since this just runs in cron. Note that this also removes the Travis configuration 🔥

**Review by pllim — APPROVED** [review]
LGTM, thanks!

As for the known failures, if we are not able to fix them in the next few weeks, maybe it is better to mark them as `xfail` for PPC platform?

**Comment by astrofrog** [comment]
@pllim - I don't think there is any harm in keeping them not xfail - to me xfail is where tests go to get ignored forever 😆

**Comment by pllim** [comment]
> just that job.

Right, but we will get weekly email about failures we have no plans to fix...

**Review by mhvk — APPROVED** [review]
Thanks so much for getting this working, @astrofrog and @olebole !

**Comment by pllim** [comment]
Thanks, all!

**Comment by Cadair** [comment]
:fire: :fire: :fire:

**Pull request merged**
